docs/cs-manual/source/tables.rst

```diff
@@ -0,0 +1,177 @@
+   ``morrowind.esm``, it does not mean the bedroll in the basement of the
+   Census and Excise Office in Seyda Neen is gone. You’re going to have to
+   delete that reference yourself or make sure that that object is replaced by
+   something that still exists otherwise you will get crashes in the worst case
+   scenario.
+
+
+
+World Screens
+*************
+
+The contents of the game world can be changed by choosing one of the options in
+the appropriate menu at the top of the screen.
+
+
+Regions
+=======
+
+This describes the general areas of Vvardenfell. Each of these areas has
+different rules about things such as encounters and weather.
+
+Name
+   This is how the game will show your location in-game.
+
+MapColour
+   This is a six-digit hexadecimal representation of the colour used to
+   identify the region on the map available in *World* → *Region Map*. If you
+   do not have an application with a colour picker, you can use your favourite
+   search engine to find a colour picker online.
+
+.. TODO  What does the last sentence above mean?
+
+Sleep Encounter
+   These are the rules for what kinds of enemies you might encounter when you
+   sleep outside in the wild.
+
+
+Cells
+=====
+
+Expansive worlds such as Vvardenfell, with all its items, NPCs, etc. have a lot
+going on simultaneously. But if you are in Balmora, why would the computer need
+to keep track the exact locations of NPCs walking through the corridors in a
+Vivec canton? All that work would be quite useless and bring your system down
+to its knees! So the world has been divided up into squares we call *cells*.
+Once your character enters a cell, the game will load everything that is going
+on in that cell so you can interact with it.
+
+In the original Morrowind this could be seen when you were travelling and you
+would see a small loading bar at the bottom of the screen; you had just entered
+a new cell and the game would have to load all the items and NPCs. The Cells
+screen in OpenMW CS provides you with a list of cells in the game, both the
+interior cells (houses, dungeons, mines, etc.) and the exterior cells (the
+outside world).
+
+Sleep Forbidden
+   Can the player sleep on the floor? In most cities it is forbidden to sleep
+   outside. Sleeping in the wild carries its own risks of attack, though, and
+   this entry lets you decide if a player should be allowed to sleep on the
+   floor in this cell or not.
+
+Interior Water
+   Should water be rendered in this interior cell? The game world consists of
+   an endless ocean at height 0, then the landscape is added. If part of the
+   landscape goes below height 0, the player will see water. (See
+   illustration.)
+
+   .. TODO  Add illustration mentioned above
+
+   Setting the cell’s Interior Water to true tells the game that this cell is
+   both an interior cell (inside a cave, for example, rather than in the open
+   air) but that there still needs to be water at height 0. This is useful for
```

Comment:
Actually, the flag only indicates that there is water if it is an interior cell. It does not make the cell interior and for exterior cells it is useless.